.travis.yml

```
@@ -197,7 +197,7 @@ jobs:
         - $( npm bin )/wp-scripts test-e2e --config=./packages/e2e-tests/jest.config.js --cacheDirectory="$HOME/.jest-cache" --runTestsByPath $( awk 'NR % 4 == 3' < ~/.jest-e2e-tests )
 
     - stage: deploy
-      if: branch = master
+      if: (NOT type IN (pull_request)) AND (branch = master)
```

Could be simpler as...

```suggestion
      if: type IN (push) AND branch = master
```

Reference: https://docs.travis-ci.com/user/conditions-v1#integration

I don't really know what `api` and `cron` are used for, but I doubt we want this stage to run for anything other than `push`.

---

`api` is for when you trigger a build through the API.

`cron` is for when a scheduled build runs.

We might have tools that trigger or schedule builds from `master` in the future and this would support that.